jekyll/_components/tables.md

```diff
@@ -5,14 +5,159 @@ order: 9
 draft: true
 ---
 
-Tabular data appears in a variety of places, the Network panel is the most obvoious example. The console is also able to output a visual table using `console.table()` e.g. `console.table(["apples", "oranges", "bananas"]);`
-
-![console table example](../images/components/tables/console-table-example.png "Console table example")
+Tabular data appears in a variety of places, the Network panel is the most obvious example. The console is also able to output a visual table using `console.table()` e.g. `console.table(["apples", "oranges", "bananas"]);`
 
 ## Usage
 
+<div class="grid-2" markdown="1">
+![Network table](../images/components/tables/table-grid-light.png)
+
+<div markdown="1">
+Tables are used when data needs to be presented in a series of rows which may also posses any amount of columns. For large quantities of data it is recommended to include a button column header allowing users to sort the data by that specific column, as seen in the Network panel
```

Comment:
Just some minor stylistic feedback: Would look nice to have periods on the end of all the sentences :). Also, this should be spelled "possess." Re: "button column header" - maybe it would be more clear to say "clickable column header"?